blogposts/code-intelligence-with-lsif.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Code intelligence with LSIF'
+author: 'Chris Wendt'
+publishDate: 2019-12-01T00:00-07:00
```

Comment:
Date is wrong, can you update?

Reply:
Oops, fixed in https://github.com/sourcegraph/about/commit/ce3e9b814c7f97f1855a2b2929741b8e9d2a1cb4